docs/sdk/client-side-sdks/javascript/javascript-gettingstarted.md

```diff
@@ -9,7 +9,7 @@ sidebar_custom_props: { icon: material-symbols:rocket }
 [![Npm package version](https://badgen.net/npm/v/@devcycle/js-client-sdk)](https://www.npmjs.com/package/@devcycle/js-client-sdk)
 [![GitHub](https://img.shields.io/github/stars/devcyclehq/js-sdks.svg?style=social&label=Star&maxAge=2592000)](https://github.com/devcyclehq/js-sdks)
 
-[//]: # (wizard-initialize-start)
+[//]: # 'wizard-initialize-start'
```

Comment:
@emmawillis this is prettier doing its thing.. should these be ignored from formatting or will this be okay for sdk install wizard?

Reply:
we shouldnt run prettier against these markdown files, I tried doing that before and it broke a bunch of functionality in the docs. These are technically `mdx` format file and not regular markdown, and prettier doesnt know what to do with them

Reply:
if thats the case, we should add a setting to not format on save explicitly on this repo

Reply:
idk how vscode does things but webstorm uses project-specific settings so mine doesn't do that